CMakeLists.txt

```diff
@@ -6,6 +6,9 @@ include(pico_sdk_import.cmake)
 # We also need PICO EXTRAS
 include(pico_extras_import.cmake)
 
+# Include example_auto_set_url
+include(example_auto_set_url.cmake)
```

Comment:
Perhaps the macro should be `playground_auto_set_url`

Reply:
Good catch! I have renamed the macro and the files where it is used to be `playground_auto_set_url` 💯